ui/lobby/css/_setup.scss

```diff
@@ -8,6 +8,7 @@ $c-slider: $c-setup;
 
   > div {
     padding: 0;
+    height: 96vh;
```

Comment:
I think you want `max-height` here. Otherwise, it will stretch on normal screens.

Reply:
Ah, great catch! Fixed in 8444828